src/applications/post-911-gib-status/sass/post-911-gib-status.scss

```diff
@@ -93,7 +93,7 @@ $formation-image-path: "~@department-of-veterans-affairs/formation/assets/img";
       a {
         text-decoration: none;
         &:after {
-          content: " (https://vets.gov" attr(href) ")";
+          content: " (https://www.va.gov" attr(href) ")";
```

Comment:
Fix for the print layout -

![image](https://user-images.githubusercontent.com/1915775/77584263-e6984c80-6eb8-11ea-8c5a-baec05d60877.png)